frontend/src/app/containers/ExperimentPage/DetailsOverview.js

```diff
@@ -0,0 +1,257 @@
+            </a>
+          </Localized>
+        </p>}
+      <dl>
+        {changelog_url &&
+          <Localized id="changelog">
+            <dt>Changelog</dt>
+          </Localized>}
+        {changelog_url &&
+          <dd>
+            <a href={changelog_url}>
+              {changelog_url}
+            </a>
+          </dd>}
+        <Localized id="contribute">
+          <dt>Contribute</dt>
+        </Localized>
+        <dd>
+          <a href={contribute_url}>
+            {contribute_url}
+          </a>
+        </dd>
+
+        <Localized id="bugReports">
+          <dt>Bug Reports</dt>
+        </Localized>
+        <dd>
+          <a href={bug_report_url}>
+            {bug_report_url}
+          </a>
+        </dd>
+
+        <Localized id="discussExperiment" $title={title}>
+          <dt>
+            {title}
+          </dt>
+        </Localized>
+        <dd>
+          <a href={discourse_url}>
+            {discourse_url}
+          </a>
+        </dd>
+      </dl>
+    </section>
+  </div>;
+
+export const ContributorsSection = ({
+  experiment: { contributors, contributors_extra, contributors_extra_url },
+  l10nId
+}: ContributorsSectionType) =>
+  <section className="contributors-section">
+    <Localized id="contributorsHeading">
+      <h3>Brought to you by</h3>
+    </Localized>
+    <ul className="contributors">
+      {contributors.map((contributor, idx) =>
+        <li key={idx}>
+          <img
+            className="avatar"
+            width="56"
+            height="56"
+            src={contributor.avatar}
+          />
+          <div className="contributor">
+            <p className="name">
+              {contributor.display_name}
+            </p>
+            {contributor.title &&
+              <Localized id={l10nId(['contributors', idx, 'title'])}>
+                <p className="title">
+                  {contributor.title}
+                </p>
+              </Localized>}
+          </div>
+        </li>
+      )}
+    </ul>
+    {contributors_extra &&
+      <p className="disclaimer">
+        <Localized id={l10nId('contributors_extra')}>
+          <span>
+            {contributors_extra}
+          </span>
+        </Localized>
+        {contributors_extra_url &&
+          <span>
+            &nbsp;
+            <Localized id="contributorsExtraLearnMore">
+              <a
+                href={contributors_extra_url}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                Learn more
+              </a>
+            </Localized>
+            .
+          </span>}
+      </p>}
+  </section>;
+
+const EXPERIMENT_MEASUREMENT_URLS = [
+  null,
+  null,
+  null,
+  'https://www.mozilla.org/privacy/websites'
+];
+
+export const MeasurementsSection = ({
+  experiment: { title, privacy_preamble, privacy_notice_url, measurements },
+  highlightMeasurementPanel,
+  l10nId
+}: MeasurementsSectionType) => {
+  const privacy = (
```

Comment:
Moved the privacy link stuff over to this sub-component from PR #2991 to resolve conflicts